andela/mmdp_site
# MMDP-530 - Edit Stakeholder directory in LGA

**What does this PR do?**

Allows an administrator to edit a selected Stakeholder

**description of Task to be completed?**

- [x] Thematic pillar count in LGA
- [x] Redirect edit stakeholder directory
- [x] LGA routes refactoring

**How should this be manually tested?**

- Check out to this branch, `ft-edit-stakeholders-in-lga-530`.
- start development server, `yarn dev`.
- Using the sidebar menu in your browser, click on the state level and click View a lga and you can click on Edit stakeholder.Make sure you have uploaded SVG Maps.


**Any background context you want to provide?**

Any pertinent information that should be considered

**What are the relevant Jira issues?**

[MMDP-530](https://viisaus.atlassian.net/browse/MMDP-530)

**Screenshots**
- None

**Questions:**

- None
